renderdoc/driver/gl/gl_debug.cpp

```diff
@@ -1975,7 +1975,8 @@ bool GLReplay::RenderTextureInternal(TextureDisplay cfg, int flags)
     ubo->Channels.x = 1.0f;
     ubo->Channels.y = 0.0f;
     ubo->Channels.z = 0.0f;
-    ubo->Channels.w = 0.0f;
+    // don't want to splat the red channel when writing depth value, therefore making alpha non-zero
+    ubo->Channels.w = dsTexMode == eGL_DEPTH_COMPONENT ? 1.0f : 0.0f;
```

Comment:
I don't quite understand this from the comment - can you explain in some more detail?